.github/REVIEWERS.yml

```diff
@@ -45,6 +48,18 @@ labels:
       - pabloem
       - Abacn
       - ahmedabu98
+      - bvolpato
+      - manavgarg
+    exclusionList: []
+  - name: spanner
+    reviewers:
+      - nielm
+    exclusionList: []
+  - name: pubsub
+    reviewers:
+      - johnjcasey
+      - manavgarg
+      - bvolpato
```

Comment:
I'd vote to remove this. This will already go to the IO team which contains all of these people and probably has sufficient expertise to effectively round robin reviews

Reply:
I think this is fair.